kitchensink/src/app/static/foundations/IconsPage.jsx

```diff
@@ -32,7 +32,7 @@ const IconsPage = () => (
           Two-tone and multi-color icons are not available in the icon font. These icons are available as SVG assets.
         </DescriptionBox>
 
-        <div className="iconGrid">
```

Comment:
Are you sure this is not used anymore?
On https://open-ch.github.io/pyrene/icons it looks like this class still adds some styles (looks like they are added globally here: https://github.com/open-ch/pyrene/blob/3ba1d1e37ebf2a7d6e249d38f632961246c716ba/kitchensink/src/css/common.css#L121) and the page looks kind of broken when I remove the class.

Reply:
Indeed, thank you for pointing that out!